### PR TITLE
ref(version): lift "unreleased" status

### PR DESCRIPTION
**Pull request by bacongobbler**
cherry-picked from the `release-3.0` branch.

DO NOT MERGE into master until Helm 3.0.0 has been released.

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>

### CODE REVIEW

**Review by thomastaylor312 — APPROVED** [review]
Good to go once 3.0.0 is out

**Comment by bacongobbler** [comment]
needs a rebase after #6868 was merged

**Review by hickeyma — APPROVED** [review]

**Comment by hickeyma** [comment]
@bacongobbler Now 3.0 is released, can this be merged?

**Pull request merged**
